src/cluster_slot_stats.c

```diff
@@ -24,6 +24,14 @@ typedef struct {
     uint64_t stat;
 } slotStatForSort;
 
+/* Struct used for storing slot statistics. */
+typedef struct slotStat {
+    uint64_t network_bytes_in;
+} slotStat;
+
+/* Struct used for storing slot statistics, for all slots owned by the current shard. */
+struct slotStat cluster_slot_stats[CLUSTER_SLOTS];
```

Comment:
This global is 128KB even in standalone mode. Can we store it inside the cluster struct so it only consumes this memory in cluster mode?